system/version.py

```diff
@@ -19,20 +19,19 @@ def cache(user_function: Callable[..., _RT], /) -> Callable[..., _RT]:
   return lru_cache(maxsize=None)(user_function)
 
 
-def run_cmd(cmd: list[str]) -> str:
-  return subprocess.check_output(cmd, encoding='utf8').strip()
+def run_cmd(cmd: list[str], cwd=None) -> str:
+  return subprocess.check_output(cmd, encoding='utf8', cwd=cwd).strip()
 
 
-def run_cmd_default(cmd: list[str], default: str = "") -> str:
+def run_cmd_default(cmd: list[str], default: str = "", cwd=None) -> str:
   try:
-    return run_cmd(cmd)
+    return run_cmd(cmd, cwd=cwd)
   except subprocess.CalledProcessError:
     return default
 
 
-@cache
```

Comment:
need an uncached version of these for the updater. I also think caching these doesn't really gain anything and can have unforseen consequences

Reply:
it prevents touching the git repo too much... one of the reasons we're moving off of git